llvm/lib/Target/AArch64/Disassembler/AArch64Disassembler.cpp

```diff
@@ -387,6 +389,20 @@ static DecodeStatus DecodeZPR2Mul2RegisterClass(MCInst &Inst, unsigned RegNo,
   return Success;
 }
 
+// Zk Is the name of the control vector register Z20-Z23 or Z28-Z31, encoded in
```

Comment:
I believe we can remove the comments now that we are not using any math to compute the register number.